postgres_ffi/build.rs

```diff
@@ -11,16 +11,19 @@ fn main() {
     // to bindgen, and lets you build up options for
     // the resulting bindings.
     let bindings = bindgen::Builder::default()
-        // The input header we would like to generate
-        // bindings for.
+        // All the needed PostgreSQL headers are included from 'pg_control_ffi.h'
         .header("pg_control_ffi.h")
+
```

Comment:
I added these newlines for clarity, to separate the different kinds of options. But `cargo fmt` insists on removing them again :-(. Any ideas?

Reply:
Idk, what about this?

```
Builder::default()
    // section one
    .foo()
    .bar()
    .baz()
    //
    // section two
    .foo()
    .bar()
    .baz()
    //
    // section three
    .foo()
    .bar()
    .baz();
```

Reply:
did it that way